static/js/controllers/appcontroller.js

```diff
@@ -58,6 +58,7 @@ define(["jquery", "angular", "underscore"], function($, angular, _) {
 				}
 			}
 		};
+		$scope.defaults.settings.sounds = playSound.getDefaults();
```

Comment:
Why is this even in the appcontroller. Put it where the other settings are initialized.